tests-e2e/core/snapshotenvironmentbinding_test.go

```diff
@@ -765,6 +779,61 @@ func buildSnapshotEnvironmentBindingStatus(components []appstudiosharedv1.Bindin
 	return status
 }
 
+// buildEnvironment creates an instance of Environment CR
+func buildEnvironment(name, displayName string) appstudiosharedv1.Environment {
+	environment := appstudiosharedv1.Environment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: fixture.GitOpsServiceE2ENamespace,
+		},
+		Spec: appstudiosharedv1.EnvironmentSpec{
+			Type:               appstudiosharedv1.EnvironmentType_POC,
```

Comment:
You can remove the EnvironmentType_POC field as it is no longer required (earlier this week we removed it from our other tests).

Reply:
Removed this line